### PR TITLE
Burials additions to existing pages

### PR DESCRIPTION
**Pull request by bethpotts**

### CODE REVIEW

**Comment by ncksllvn** [comment]
```
Error: You have 3 broken links:
       
       burials-memorials/eligibility/index.html >>> href: "/burials-and-memorials/honor/", text: "Learn about memorials and burial honors"
       burials-memorials/eligibility/index.html >>> href: "/burials-and-memorials/survivor-and-dependent-benefits/", text: "Learn about survivor benefits"
       burials-memorials/eligibility/index.html >>> href: "/burials-and-memorials/pre-need/", text: "Find out how to apply"
```

File: pages/burials-memorials/eligibility.md [hunk 1]
```diff
@@ -65,21 +68,35 @@ At the time of need, the person planning the burial can also request other honor
 - [Presidential Memorial Certificate signed by the current President of the United States](/burials-memorials/memorial-items/presidential-memorial-certificates/)
 - [Headstone, marker, niche cover, or medallion to identify a gravesite located outside a VA national cemetery](/burials-memorials/memorial-items/headstones-markers-medallions/)
 
+
 [Learn about the burial planning process](/burials-memorials/plan-a-burial/). <br />
-[Learn about memorial items](/burials-memorials/memorial-items/).
+[Learn about memorials and burial honors](/burials-memorials/honor/). <br>
```
Comment (ncksllvn) [inline pages/burials-memorials/eligibility.md]: Does `/burials-memorials/honor/` exist? It's being reported as broken.

File: pages/burials-memorials/eligibility.md [hunk 2]
```diff
@@ -65,21 +68,35 @@ At the time of need, the person planning the burial can also request other honor
...
+[See our National Cemetery Administration brochure](https://www.cem.va.gov/cem/docs/factsheets/NCA_Brochure.pdf).
+
 
 ## If I'm the survivor of a Veteran or Servicemember, what other benefits can I get?
 
-We give the surviving spouse, children, and parents of Veterans and Servicemembers a range of benefits. These may include tax-free monetary benefits (called Dependency and Indemnity Compensation) and help paying for the funeral service (called a burial allowance).<br>
-[See if you're eligible for Dependency and Indemnity Compensation (DIC)](/burials-memorials/dependency-indemnity-compensation/). <br>
-[Find out how to apply for a burial allowance](/burials-memorials/veterans-burial-allowance/).
+We give the surviving spouse, children, and parents of deceased Servicemembers and Veterans a range of benefits. These may include tax-free monetary benefits and help paying for the funeral service. <br>
+[Learn about survivor benefits](/burials-memorials/survivor-and-dependent-benefits/).
```
Comment (ncksllvn) [inline pages/burials-memorials/eligibility.md]: I don't think `/burials-memorials/survivor-and-dependent-benefits.md` exists either.

**Comment by ncksllvn** [comment]
@bethpotts I resynced this branch with master to resolve the errors you were seeing. This PR has been around quite awhile!

**Comment by bethpotts** [comment]
@ncksllvn I saw your note above about this PR being around for a while. Yes, we're still making changes in this branch. The goal is to wrap this up by end of May. There's still work to do. What I'd like to know is if my thinking re: adding the style guide updates for burials to this PR is a sound idea. As far as number of commits, it'll make this PR larger. Does that matter? I wanted the style guide updates to be done in this branch so as not to open another branch for that work off master that will have a lot of conflicts with this old PR. 

Is there any other option you want to suggest for getting these two streams of work done? I paused the copyeditor who had just started on style guide updates in this branch/PR to ask you if you think it's ok for her to proceed. Thanks!

**Comment by ncksllvn** [comment]
@bethpotts Thank you for checking with me! But however works best for you is totally fine, just as long we make sure to resolve conflicts soon after they come about. In the end, as long as the "Files changed" tab looks okay, then we should be just fine. Sometimes when a PR is super big it's hard to do the final review, but as long as you're comfortable then it works for me!

**Comment by peggygannon** [comment]
@Karak888 @ncksllvn fyi, this is the PR/heroku instance from which  the CMS team will be migrating Burials to drupal. 

This PR contains a bunch of edits/rewrites to Burials (that are still awaiting SME approval). 

cc @peggygannon

**Comment by AnneHurley** [comment]
Check H1s for character count limit #19036

@peggygannon @jenniferlee-dsva
Shorter H1s with character counts at 50 or less:

Benefits for spouse and dependents (VA DIC)
Honor a Veteran or Reservist with memorial items
Schedule a burial for a Veteran or family member
Pre-need eligibility for burial in a VA cemetery
After you apply for an eligibility determination

**Comment by jenniferlee-dsva** [comment]
@AnneHurley @peggygannon @Karak888  - Just a reminder: Since all Burials pages are now in CMS migration QA content freeze, please be sure to create a ticket/epic to ensure that changes on Github .md files get manually made to the Drupal instance post-migration freeze.

**Comment by peggygannon** [comment]
@jenniferlee-dsva Yes, will do! I'll create an issue to check these titles, post-migration.

**Comment by timwright12** [comment]
closing due to age an inactivity

**Pull request closed without merging**